spec/Classes.tex

```diff
@@ -106,7 +108,9 @@ \subsection{Class Types}
 \end{syntax}
 
```

Comment:
I'd rather avoid repetition in the following. For example:

A class type, including a generic class type that is not fully instantiated, may appear in inheritance lists of other class declarations.

Also, is "fully specified" a legit term here? I am proposing "fully instantiated" instead.